Feature - 3076 Highlight sorted columns in the UI

Poltergeist opened the pull request:
Close mesosphere/marathon#3076

pierlo-upitup commented on src/css/layout/table.less:
```diff
@@ -21,7 +21,9 @@
   border: none;
   background-color: @table-header-color;
-
+  &.table-column-sorted{
```
How about calling this class simply `highlight-column`? We are already in the `table` domain, so why repeat it. Also, `sorted` does not really reflect well what the class does, imho.

Either way, it needs a space before the curly ;)

pierlo-upitup commented on src/test/units/AppListItemComponent.test.js:
```diff
@@ -51,4 +51,55 @@ describe("AppListItemComponent", function () {
+            sortKey={testCase.sortKey} />
+        );
+
+        expect(component.find(testCase.cssClass).get(0).attribs.class)
```
you can use `.first()` as well ;)

pierlo-upitup commented:
Just my opinion on the looks of this: I'm not a fan darkening all columns by default, and to only highlight the sorted column. By default, the largest column is highlighted, and it looks rather weird:

<img width="1126" alt="screen shot 2016-01-28 at 16 30 24" src="https://cloud.githubusercontent.com/assets/1078545/12648952/b104511a-c5dc-11e5-95c3-c7c771942ff0.png">

@leemunroe have you considered a different solution, where we restore the default table background color so it sticks out from the page background and lighten up the sorted column from there?

leemunroe commented:
Cool to see this in action. Agree with @pierlo-upitup that our color choice looks odd, especially on the big column.

What if we make it a bit more subtle?
* Change `th` background to `rgba(255, 255, 255, 0.06)`
* Change `td` background to `rgba(255, 255, 255, 0.01)` 

![image](https://cloud.githubusercontent.com/assets/15963/12668717/54c61ec4-c60e-11e5-9c0a-a82b69114c42.png)

Also I think on row hover I would also expect the full row to still change to the same background color.

![image](https://cloud.githubusercontent.com/assets/15963/12668745/97a545b2-c60e-11e5-858f-2d4476f154eb.png)

leemunroe commented:
Something else to consider, which may also help address @pierlo-upitup's concern.

We played with padding and headings. Helps the table stand out a bit more. And also a small step towards DCOS alignment. We can have a separate discussion about it from this PR. 

![image](https://cloud.githubusercontent.com/assets/15963/12669023/7ae0dcfe-c611-11e5-87d3-ee539d9fe6bc.png)

pierlo-upitup commented:
Yeah, agreed - let's move the padding adjustment to a separate PR/Issue. I like the suggested color scheme much better!

Poltergeist commented:
Changed to color of the columns according to leemunroe's suggestions

leemunroe commented:
:shipit:

orlandohohmeier commented:
![](https://cloud.githubusercontent.com/assets/647035/12759095/b3314e1e-c9df-11e5-8534-b5f4a58addd6.png)
The instance/task list status column highlighting does not work. 

*BTW: Please excuse my very late reply, I didn't manage to test your PR until today. :disappointed:*

Poltergeist commented:
@orlandohohmeier I rebased master and fixed the status column. Thank you for reviewing this PR so good and finding the bug. Better late and complete then early and incomplete.
